core-common/src/main/jsr166/org/glassfish/jersey/internal/jsr166/Flow.java

```diff
@@ -1,3 +1,19 @@
+/*
```

Comment:
This file has CC0 license. We must not put EPL in this file.

Reply:
core-common/src/main/jsr166/org/glassfish/jersey/internal/jsr166/ should by skipped if I get this right

Reply:
This was excluded already /core-common/src/main/java/org/glassfish/jersey/internal/jsr166
So I have removed the copyright on that file and I added:
/core-common/src/main/jsr166/org/glassfish/jersey/internal/jsr166